packages/lexical-editor/src/nodes/webinyNodes.ts

```diff
@@ -62,5 +62,20 @@ export const WebinyNodes: ReadonlyArray<
         with: () => {
             return new QuoteNode();
         }
+    },
+    LinkNode,
```

Comment:
Not related that much to this PR but... 

I'm still not 100% sure on why in this list we're first registering e.g. `LinkNode`, and then registering the replacement of all `BaseLinkNode` with `LinkNode`. 

Can you remind me... The base nodes are already registered somewhere in Lexical lib's internals? That's why we don't see the registration of base nodes like `BaseLink` here, right?

Reply:
We need to register our node as the default node for the link. We must provide only one node for the link, and our custom node is now in charge of the links. 

When lexical is traversing and find the built-in `link` node, checks if we have registered a custom default node, then on the `with: (node: BaseLinkNode)` line, will try to transfer the constructor data and create an instance of our custom `LinkNode` with type` link-node`. 

In this way, lexical has a mechanism to support old and new links. The old type `link` will be converted to the custom type `link-node`.

Reply:
All good. Just wasn't sure about the "The base nodes are already registered somewhere in Lexical lib's internals?" part.

Let's add a comment about this just for future, and we can merge this (once we release 5.37.1).